ui-components/src/headers/SiteHeader.tsx

```diff
@@ -1,20 +1,23 @@
-import * as React from "react"
+import React, { useState } from "react"
 import { LocalizedLink } from "../actions/LocalizedLink"
 import { LanguageNav, LangItem } from "../navigation/LanguageNav"
 
 export interface SiteHeaderLanguage {
   list: LangItem[]
   codes: string[]
 }
+type LogoWidth = "slim" | "medium" | "wide"
```

Comment:
Since we were just talking about interviews, this made me think of a good question. Since Typescript supports unions on types, what are the advantages/disadvantages of using this over an enum? But that seems a little too specific.

Reply:
Interesting q! I'm actually not sure what I would say, knowing that both get Typescript autocomplete. After a google I see that enums are....enumerable (hehe) but other than that I wouldn't be able to articulate other pros/cons.